launch_testing/launch_testing/pytest/hooks.py

```diff
@@ -160,7 +160,7 @@ def collect(self):
         if _pytest_version_ge(7):
             # self.path exists since 7
             from _pytest.pathlib import import_path
-            module = import_path(self.path, root=None)
+            module = import_path(self.path, root=None, consider_namespace_packages=False)
```

Comment:
So as it stands, this isn't going to work properly with all versions of `pytest`.

In particular, version 7.0.0 up to and including version 8.0.2 required this argument to *not* be there; it is only since 8.1.0 that this is required.  Thus, I think this whole block needs to be reworked a bit, something like:

```
if _pytest_version_ge(8, 1, 0):
    from _pytest.pathlib import import_path
    module = import_path(self.path, root=None, consider_namespace_packages=False)
elif _pytest_version_ge(7, 0, 0):
    from _pytest.pathlib import import_path
    module = import_path(self.path, root=None)
else:
    module = self.fspath.pyimport()
```

The same goes for the one below.  Maybe we should just make a common function for this, while we are at it.

Reply:
Good point. Changes made. I did it without making a function for it if that's also fine with you. My threshold to making a function out of a piece of code is usually 3 if it's small enough :sweat_smile: but I can do it if you feel strongly about it

Reply:
> My threshold to making a function out of a piece of code is usually 3 occurrences if it's small enough 😅 but I can do it if you feel strongly about it

My experience here with other python dependencies is that we'll need to add more of these blocks in the future, so I figured we can just put it in one place.  But I don't feel strongly about it, so we can leave it as-is for now.